transactions/epoch/admin/update_epoch_config.cdc

```diff
@@ -0,0 +1,22 @@
+import FlowEpoch from 0xEPOCHADDRESS
+
```

Comment:
I would also note in the documentation that we assume here that the DKG phase length is much smaller than the staking length for all potential configurations. Otherwise we would need to do the same if/else check for deciding whether to update the DKG length before the staking length.

Reply:
https://github.com/onflow/flow-core-contracts/pull/282/commits/c58ed63eddd13fdacde845bdf7120038bff021ad